concordium-consensus/tests/scheduler/SchedulerTests/SmartContracts/V1/Checkpointing.hs

```diff
@@ -343,7 +546,7 @@ deploymentCostCheck _ Types.TransactionSummary{..} = do
   where
     checkSuccess msg Types.TxReject{..} = assertFailure $ msg ++ show vrRejectReason
     checkSuccess _ _ = return ()
-    
+
 tests :: Spec
 tests = describe "V1: Checkpointing." $
-  mkSpecs $ testCases1 ++ testCases2 ++ testCases3 ++ testCases4
+  mkSpecs $ testCase1 ++ testCase2 ++ testCase3 ++ testCase4 ++ testCase5 ++ testCase6
```

Comment:
It looks like all of these test cases are singleton lists. So it would probably be a tiny bit cleaner if they had type `TestCase` and then just construct a list here.